File: BackEndServices/BackEndServices/Sensor/Sensor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BackEndServices.Sensor
{
   public class Sensor : ISensor
    {
        public int SensorID { get; private set; }

        public string SensorName { get; private set; }


        public Sensor(int sensorID,string sensorName)
        {
            SensorID = sensorID;

            SensorName = sensorName;
        }
    }
}

File: Angular/world-goals-app/src/app/sensor.ts
export class Sensor {
    /**
     *
     */
    sensorID : number;
    sensorName : string;

    
    constructor(sensorID : number, sensorName : string) {
        this.sensorID = sensorID;

        this.sensorName = sensorName;
    }
}
